app/src/editor/shared/components/modules/ExportCSV.pcss

```diff
@@ -0,0 +1,21 @@
+.container {
+  padding: 0 24px;
+}
+
+.text {
+  font-family: 'IBM Plex Mono', monospace;
+  font-size: 10px;
+  font-weight: 500;
+  color: #525252;
+  line-height: 2em;
+}
+
+.text a {
+  text-decoration: underline;
+  color: #525252;
```

Comment:
you could make these anchor styles `color: inherit` so it picks up the colour defined in `.text`